### ISSUE TITLE
ModuleNotFoundError version 0.2.0

### ISSUE THREAD

**Issue by pfreiberger** [MichaelKavanagh/elexon#8]
Although, it is possible to isntlal the package it is not usable anymore. The module can be found via pip freeze but raises a ModuleNotFoundError when trying to install. The previous version 0.2.0 still works.
```
(test_env) C:\Users\user\test>pip install elexon
Looking in indexes: https://pypi.python.org/simple/, 
Collecting elexon
  Downloading elexon-0.2.2-py3-none-any.whl (3.0 kB)
Installing collected packages: elexon
Successfully installed elexon-0.2.2
WARNING: You are using pip version 20.1.1; however, version 21.2.4 is available.
You should consider upgrading via the 'c:\users\user\test\test_env\scripts\python.exe -m pip install --upgrade pip' command.

(test_env) C:\Users\user\test>python
Python 3.8.5 (default, Sep  3 2020, 21:29:08) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import elexon
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'elexon'


(test_env) C:\Users\user\test>pip freeze
elexon==0.2.2
```

**Comment by michaelkavanagh** [comment]
Rats, looks like I broke it with the change from setup.py to setup.cfg.

I've just reverted the commit, seems to work for me. Please could you test to confirm? Thanks.

**Comment by pfreiberger** [comment]
I can confirm that it is working again after upgrading to 0.2.3

**Comment by michaelkavanagh** [comment]
Great, thanks.
Fixed by 14ce34c64ee7e2a4fb2eadb79e0954718fdf1055
